### PR TITLE
[Requirements] Bump storey to 1.8.0, add test

### PR DESCRIPTION
**Pull request by gtopper**
[ML-7818](https://iguazio.atlassian.net/browse/ML-7818)

[ML-7818]: https://iguazio.atlassian.net/browse/ML-7818?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/feature-store/test_steps.py [hunk 1]
```diff
@@ -733,6 +734,59 @@ def test_imputer_default_value(rundb_mock, engine):
     assert not imputed_df.isnull().values.any()
 
 
+class MyChoice(storey.Choice):
+    def select_outlets(self, event):
```
Comment (Eyal-Danieli) [inline tests/feature-store/test_steps.py:738]: just a suggestion
```suggestion 
    @staticmethod
    def select_outlets(event):
```

**Review by Eyal-Danieli — APPROVED** [review]
LGRM
just a single suggestion

File: tests/feature-store/test_steps.py (hunk 1 above)
Comment (gtopper) [reply tests/feature-store/test_steps.py:738]: `select_outlets()` is an instance method in `storey.Choice`. It's not good practice to override an instance method with a static method. It also causes static analysis warnings due to the signature not matching.

**Review by guy1992l — COMMENTED** [review]
Looks good to me 👍

**Pull request merged**